### PR TITLE
remove the redundant part in quick start in tftraining

### PR DESCRIPTION
**Pull request by ChanYiLin**
I think this "Submitting a TensorFlow training job" part is redundant in quick start.
We should remove it.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/147)

### CODE REVIEW

**Comment by ChanYiLin** [comment]
Hi would you please help me to have a look at this commit?
assign @pdmack

**Comment by ChanYiLin** [comment]
Hi would you please help me to have a look at this commit?
/assign @pdmack

**Comment by ChanYiLin** [comment]
Hi @texasmichelle , thanks!
/assign @texasmichelle

**Comment by ChanYiLin** [comment]
/assign @jlewi 
Hi, I think we should fix this.
Could you help me to check it? Thanks!

File: content/docs/guides/components/tftraining.md [hunk 1]
```diff
@@ -204,70 +204,6 @@ Typically you will want to change the following values
...
-Kubeflow ships with a [ksonnet prototype](https://ksonnet.io/docs/concepts#prototype) suitable for running the [TensorFlow CNN Benchmarks](https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks).
-
-You can also use this prototype to generate a component which you can then customize for your jobs.
-
-Create the component
-
-```
-CNN_JOB_NAME=mycnnjob
-
-ks registry add kubeflow-git github.com/kubeflow/kubeflow/tree/${VERSION}/kubeflow
```
Comment (jlewi) [inline content/docs/guides/components/tftraining.md]: Where else in the docs is this information provided?

**Comment by jlewi** [comment]
Can you provide a link to the other parts of the docs where you think this information is already provided?

**Comment by ChanYiLin** [comment]
@jlewi 
sorry is my fault, I didn't make it clear.
this "Submitting a TensorFlow training job" is actually repeated on the same page.
First starts from line 143 to 206, the repeated one starts from line 207 to 270 which are removed by this PR.

File: content/docs/guides/components/tftraining.md (hunk 1 above)
Comment (ChanYiLin) [reply content/docs/guides/components/tftraining.md]: this information is already there started from line 143, whereas this part from line 207 is repeated.

**Comment by jlewi** [comment]
Thanks; sorry I missed it.

/lgtm
/approve

**Pull request merged**